config.json

```
@@ -19,16 +19,16 @@
   "test_pattern": ".*\\.rakutest$",
   "files": {
     "solution": [
-      "%{kebab_slug}.rakumod"
+      "%{pascal_slug}.rakumod"
     ],
     "test": [
       "%{kebab_slug}.rakutest"
     ],
     "example": [
-      "%{kebab_slug}.rakumod"
+      ".meta/solutions/%{pascal_slug}.rakumod"
     ],
     "exemplar": [
-      "%{kebab_slug}.rakutest"
+      ".meta/solutions/%{kebab_slug}.rakutest"
```

```suggestion
      ".meta/solutions/%{kebab_slug}.rakumod"
```

I think this should be `.rakumod`. Exemplar is "the idiomatic, one true exemplary implemetentation"

---

Ah, I think I've misunderstood what exemplar is for. I'm not sure if its applicable for this track? Is there an example of this type of file on another track?

---

It's applicable if you have concept exercises. 

`exemplar`: https://github.com/exercism/javascript/blob/main/exercises/concept/lasagna/.meta/exemplar.js
Meant to be a canonical idiomatic solution for **concept** exercises only.

`example`: https://github.com/exercism/javascript/blob/main/exercises/practice/accumulate/.meta/proof.ci.js
Meant to be a proof that the exercise is solvable.

---

Ah I see, thanks for the info!